### PR TITLE
Fix old content migrations' database state

### PR DESCRIPTION
**Pull request by xamanu**
PR together with @ramyaragupathy.

In some old migrations we spotted some lines that were causing errors today, like on #1487:

`sqlalchemy.exc.ProgrammingError: (psycopg2.ProgrammingError) column projects.mapping_editors does not exist`

This was caused by running content migrations on the database in these files, but not looking at the respective state of the database (schema) but rather pulling in the latest state from code with:

```
projects = Project.__table__	
tasks = Task.__table__
messages = Message.__table__
``` 

This caused errors for missing columns that have been created in later migrations. We replaced it with direct queries to the database which will return in result sets with the database schema as it is at the moment of running the migration.

### CODE REVIEW

**Comment by xamanu** [comment]
There is a follow up issue, which you might want to test together #1548.

**Comment by xamanu** [comment]
Cool. Please squash your commits, too. Thanks.

**Comment by ethan-nelson** [comment]
Can't run the tests locally, but it makes sense to me: we shouldn't rely on the structure presently defined in the models file, but rather on the structure in the database prior to the migration.

File: migrations/versions/0a6b82b55983_.py [hunk 1]
```diff
@@ -78,9 +78,15 @@ def upgrade():
 
             # Only process messages from projects that still exist
             if project_existence[project_id]:
-                op.execute(
-                    messages.update().where(messages.c.id == message.id)
-                        .values(message_type=message_type, project_id=project_id, task_id=task_id))
+                if task_id is None:
```
Comment (nrotstan) [inline migrations/versions/0a6b82b55983_.py]: This check and change to 'null' is also needed for message_type and project_id.

**Comment by ramyaragupathy** [comment]
Thanks for the review @nrotstan. I noticed this failing on another branch and was digging into what was going wrong. This helps!

**Comment by xamanu** [comment]
Tested again today with @ramyaragupathy. And moving in.

**Pull request merged**